cmd/protecodeExecuteScan.go

```diff
@@ -336,5 +336,12 @@ func getTarName(config *protecodeExecuteScanOptions) string {
 	}
 	// replace unwanted chars
 	fileName = strings.ReplaceAll(fileName, "/", "_")
+	log.Entry().Infof("Filename : %s", fileName)
+	input := "@sha256"
+	if strings.Contains(fileName, input) { 
+		fileName = strings.ReplaceAll(fileName, ":",    "_")
+		fileName = strings.ReplaceAll(fileName, "@sha256", "")
+		log.Entry().Infof("Image Pulled via sha: %s", fileName)
```

Comment:
> `Image Pulled` ...

Looks misplaced. In the context of this function here (`getTarName`) nothing has been pulled. We should not make assumptions about when/how the function is called. I would like to suggest to either rephrase the log message or to move that log message right after the point in time when the download really happened.

Reply:
misplaced log removed